src/java/org/apache/poi/ss/usermodel/Column.java

```diff
@@ -0,0 +1,333 @@
+package org.apache.poi.ss.usermodel;
+
+import java.util.Iterator;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.util.NotImplemented;
+
+/**
+ * A read-only Row view of a sheet column.
+ * 
+ * Intended to iterate over cells of a sheet.
+ *
+ * Get this object with Sheet.getColumn(colNum).
+ *
+ * @author luca vercelli 2018
+ */
+public class Column implements Row {
```

Comment:
`Row` shouldn't be implemented because a column is obviously not a row. On account of implementing this interface there are various methods in this class which don't make any sense

```java
    @Override
    public int getRowNum() {
        return colNum;
    }

    @Override
    @NotImplemented
    public short getHeight() {
        throw new IllegalStateException("Not implemented");
    }

    @Override
    @NotImplemented
    public float getHeightInPoints() {
        throw new IllegalStateException("Not implemented");
    }
```